zerver/lib/digest.py

```diff
@@ -202,6 +210,9 @@ def get_recent_topics(
 def get_hot_topics(
     all_topics: List[DigestTopic],
     stream_ids: Set[int],
+    use_new_ranking_system: bool = False,
+    message_weight: Optional[int] = None,
+    sender_weight: Optional[int] = None,
```

Comment:
Do we need the `use_new_ranking_system` parameter?  It seems like it could be inferred by the presence of the `message_weight` / sender_weight` parameters.

Reply:
(Or perhaps we should pass a single `Optional[DigestWeights]` dataclass object containing both fields as `int` type.

Reply:
Sure .